google/ggrc-core
# GGRC-6298 Move IssueTracker default values to environment variables

# Issue description

 Issue tracker default values are kept both on FE and BE side as hardcoded values. 

# Steps to test the changes

1. Set environment variables(`ISSUETRACKER_ISSUE_COMPONENT_ID` for example)
2. Check that variable in `DEFAULT_ISSUETRACKER_VALUES ` dict is exist

# Solution description

Moved issue tracker default values to environment variables 

# Sanity checklist


# PR Review checklist
